### DISCUSSION TITLE
How can I login to snappymail with NC account

### DISCUSSION THREAD

**Discussion by eugenov** [the-djmaze/snappymail#945] [Q&A]
with no email connected to it. Just NC username and password. Then I could connect email accounts.
Right now https://github.com/the-djmaze/snappymail/issues/569#issuecomment-1426209304

**Comment by the-djmaze** [comment]
Nextcloud:
- username: john
- password: secret

IMAP/SMTP:
- username: john
- password: secret

Then you can use the Nextcloud -> Admin -> Additional settings option to autologin.
In SnappyMail you need:
- domain `nextcloud` and default host `nextcloud` but then sending mail might fail
- domain `example.com` default host `exame.com`

What do you want as setup?
